apps/server/src/migrations/mikro-orm/Migration20250708112356.ts

```diff
@@ -0,0 +1,137 @@
+import { Migration } from '@mikro-orm/migrations-mongodb';
+
+const config = {
+	roomviewer: {
+		old: ['ROOM_VIEW', 'ROOM_LEAVE'],
+		new: ['ROOM_LEAVE_ROOM', 'ROOM_LIST_CONTENT'],
+	},
+	roomeditor: {
+		old: ['ROOM_VIEW', 'ROOM_CONTENT_EDIT', 'ROOM_LEAVE'],
+		new: [
+			'ROOM_EDIT_CONTENT',
+			'ROOM_LEAVE_ROOM',
+			'ROOM_LIST_CONTENT',
+			'ROOM_LIST_DRAFTS',
+			'ROOM_MANAGE_VIDEOCONFERENCES',
```

Comment:
that should not be here, as the feature we implemented for the video conferences otherwise makes no sense, did not have time yet to check the remaining PR